integrations/oci/tls-certificates/src/main/java/io/helidon/integrations/oci/tls/certificates/DefaultOciCertificatesTlsManager.java

```diff
@@ -83,35 +81,30 @@ class DefaultOciCertificatesTlsManager extends ConfiguredTlsManager implements O
     }
 
     @Override // TlsManager
-    public void init(Tls tls) {
-        // this will establish whether we are enabled or not
-        super.init(tls);
-
-        // if we are enabled then we will setup the ssl context, but first we will need to download it
-        if (tls.enabled()) {
-            try {
-                Services services = InjectionServices.realizedServices();
-                this.pkDownloader = services.lookupFirst(OciPrivateKeyDownloader.class);
-                this.certDownloader = services.lookupFirst(OciCertificatesDownloader.class);
-                this.asyncExecutor = Executors.newSingleThreadScheduledExecutor();
-                this.async = Async.builder().executor(asyncExecutor).build();
-
-                // the initial loading of the tls
-                maybeReload();
-
-                // now schedule for reload checking
-                String taskIntervalDescription =
-                        io.helidon.scheduling.Scheduling.cronBuilder()
-                                .executor(asyncExecutor)
-                                .expression(cfg.schedule())
-                                .task(inv -> maybeReload())
-                                .build()
-                                .description();
-                LOGGER.log(System.Logger.Level.DEBUG, () ->
-                        OciCertificatesTlsManagerConfig.class.getSimpleName() + " scheduled: " + taskIntervalDescription);
-            } catch (Exception e) {
-                throw new RuntimeException("Unable to initialize", e);
-            }
+    public void init(TlsConfig tls) {
+        this.tlsConfig = tls;
+        try {
+            Services services = InjectionServices.realizedServices();
```

Comment:
i would prefer to only realize these services if we are enabled as it was prior.

Reply:
check the javadoc, this method is never called if the TLS is disabled.